lib/ZcapClient.js

```diff
@@ -140,6 +140,9 @@ export class ZcapClient {
     capability, controller, invocationTarget, expires,
     allowedActions = []
   } = {}) {
+    if(!controller) {
+      throw new TypeError('"controller" must be provided.');
```

Comment:
```suggestion
      throw new TypeError('"controller" is required.');
```
Seems more consistent with the language here: https://github.com/digitalbazaar/ezcap/blob/9fb30bc79fea622c0397a9f69ea4d34b1563a352/lib/ZcapClient.js#L158